tests/v2/test_waku_filter.nim

```diff
@@ -128,3 +129,30 @@ procSuite "Waku Filter":
     check:
       # Check that unsubscribe works as expected
       (await responseCompletionFuture.withTimeout(5.seconds)) == false
+  
+  asyncTest "handle filter subscribe failures":
+    const defaultTopic = "/waku/2/default-waku/proto"
+
+    let
+      key = PrivateKey.random(ECDSA, rng[]).get()
+      peer = PeerInfo.init(key)
+      contentTopic = ContentTopic(1)
+      post = WakuMessage(payload: @[byte 1, 2, 3], contentTopic: contentTopic)
```

Comment:
I think `key`, `peer` and `post` are not used in the test, you mat remove them.

Reply:
Thanks, Sanaz. Done in 4e7f068